data/core/statusview.lua

```diff
@@ -1200,6 +1224,19 @@ function StatusView:draw()
       end
     end
   end
+
+  if clicks > 5 then
+    core.root_view:defer_draw(function()
+      local font = type(config.stonks) == "table" and config.stonks.font or style.icon_font
+      local icon = type(config.stonks) == "table" and config.stonks.icon or ( config.stonks and "g" or "h" )
+      local xadv = renderer.draw_text(font, icon, gx, gy, style.text)
```

Comment:
Needs more RGB for faster performance.

Reply:
Will fix.

Reply:
From my testing the commit improved performance by another 500%.